CHANGELOG-1.1.md

```diff
@@ -1,3 +1,21 @@
+# TiDB Operator v1.1.4 Release Notes
+
+## Notable changes
+
+- `TableFilter` is added to the `BackupSpec` and `RestoreSpec`, which supports backup specific database or table with Dumpling or BR and supports restore specific database or table with BR.
+  `BackupSpec.Dumpling.TableFilter` is deprecated from v1.1.4 and please configure `BackupSpec.TableFilter`.
+  From TiDB v4.0.3, you can configure `BackupSpec.TableFilter` to replace the `BackupSpec.BR.DB` and `BackupSpec.BR.Table` fields and configure `RestoreSpec.TableFilter` to replace the `RestoreSpec.BR.DB` and `RestoreSpec.BR.Table` fields ([#3134](https://github.com/pingcap/tidb-operator/pull/3134), [@sstubbs](https://github.com/sstubbs))
```

Comment:
```suggestion
  Since TiDB v4.0.3, you can configure `BackupSpec.TableFilter` to replace the `BackupSpec.BR.DB` and `BackupSpec.BR.Table` fields and configure `RestoreSpec.TableFilter` to replace the `RestoreSpec.BR.DB` and `RestoreSpec.BR.Table` fields ([#3134](https://github.com/pingcap/tidb-operator/pull/3134), [@sstubbs](https://github.com/sstubbs))
```